ebusd/config.yaml

```diff
@@ -14,27 +14,43 @@ arch:
   - i386
 init: false
 startup: "system"
+services:
+ - mqtt:need
```

Comment:
i am going to remove needs MQTT.  I think most people will run with MQTT but it is not mandatory

Reply:
Or change to "want" ? from the Documentation: want (this add-on can use this service)

Reply:
sound good - if you can't yet see my other comments (I submitted my review wich I think should have published my comments?)

"wants" might be the correct option here if many people opt to use MQTT